### ISSUE TITLE
Cleanup accessibility text

### ISSUE THREAD

**Issue by LuLaValva** [eBay/evo-web#54]
From discussion with Ian... we should look through all of our components and figure out which ones have English defaults for accessibility text. WAE won't catch issues if there is _something_ there, even if that something is wrong.

EDIT: Goutham and I decided to expand on this ticket for some additional cleanup related to a11y.

## List a11y attributes with bad names:

* button
   * a11yText | aria label for button when bodyState === loading
      * should be a11yLoadingText 

## List a11y attributes that are missing defaults:

* button
   * a11yText | aria label for button when bodyState === loading
      * currently empty, should be "Loading" 
      * 
## List a11y attributes that have bad defaults

tbd

## List a11y attributes that have bad storybook example text

* carousel
   * aria-label = "Continious"
      * Should be empty

**Comment by ArtBlue** [comment]
@saiponnada , this seems like the perfect task for Claude. Can we leverage that to do a full sweep??

**Comment by ArtBlue** [comment]
Team agreed to close this because it's being addressed in the new evo components. We also made a lot of improvements already in the current components.
